charts/ztka/templates/_helpers.tpl

```diff
@@ -213,3 +221,16 @@ Get user full-qualified domain.
 {{- define "ztka.userFQDN" -}}
 {{.Values.fqdn.userSubdomain}}.{{.Values.fqdn.domain}}
 {{- end -}}
+
+{{/*
+Get paralus service with port that is endpoint for kratos after login webhook.
+*/}}
+{{- define "ztka.afterLoginWebhookWithPort" -}}
+{{ $url := "http://a.aaa"}}
```

Comment:
do we really need to default it? if so use localhost:11000 instead of this

Reply:
Just for safety. Will use default: localhost:11000 .